src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java

```diff
@@ -217,13 +217,7 @@ protected boolean updateStockWardQuantity(
             if(medicalWardTo != null) {
             	repository.updateInQuantity(Math.abs(qty), wardTo, medical, lot);
             } else {
-				MedicalWard medicalWard = new MedicalWard();
-				medicalWard.setWard(movement.getWard());
-				medicalWard.setMedical(movement.getMedical());
-				medicalWard.setInQuantity((float)Math.abs(qty));
-				medicalWard.setOutQuantity(0.0f);
-				medicalWard.setLot(movement.getLot());
-                repository.save(medicalWard);
+                repository.insertMedicalWard(wardTo, medical, Math.abs(qty), lot);
```

Comment:
Is it not the equivalent of save(medicalWard) with correct parameters set?

Reply:
Should be but reverting to insertMedicalWard made tests working again. I'll add a comment to investigate this and restore save

Reply:
And I see that you switched the ui so two questions:
1. Should I care about old swing based ui? I have quick fix for menu issue after jpa changes in separate pull request
2. How to test changes now? 

And we definitely need to test everything quite deeply. There are many contributors and whole OP-128 is quite big change

Reply:
1. yes, until the "ui" is not yet ready
2. if you mean test from the "gui", I think smoke-tests and end-users is the way

I agree, I suggest all contributors to fork also "gui" and to have the application up and running for both regression tests and help on the new "ui" design